docs/stdlib/json.rst

```diff
@@ -289,20 +285,20 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
     :index: array unpack
 
-    Return elements of JSON array as a set of :eql:type:`json`.
+    Returns the elements of a JSON array as a set of :eql:type:`json`.
 
-    Calling this function on anything other than a JSON array will
-    cause a runtime error.
-
-    This function should be used if the ordering of elements is not
-    important or when set ordering is preserved (such as an immediate
-    input to an aggregate function).
+    Calling this function on anything other than an array of :eql:type:`json`
+    will result in a runtime error:
 
     .. code-block:: edgeql-repl
 
         db> select json_array_unpack(to_json('[1, "a"]'));
         {'1', '"a"'}
 
+    This function should be only used if the ordering of elements are not
+    deemed important, or when the ordering of the set is preserved (such as an
+    immediate input to an aggregate function)
+
```

Comment:
This should be restored to its previous location. See previous comment for details.